examples/00-postprocessing/post_processing.py

```diff
@@ -20,32 +20,38 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-""".. _ref_updated_exhaust_manifold_example:
+""".. _ref_post_processing:
 
-Enhanced Postprocessing with PyVista and Matplotlib
----------------------------------------------------
-This updated example demonstrates postprocessing capabilities in PyFluent
-using an object-oriented approach, providing a more user-friendly interface
-and improved flexibility.
-The 3D model used in this example is an exhaust manifold, where high-temperature
-turbulent flows are analyzed in a conjugate heat transfer scenario.
+Post-processing using Pyvista and Matplotlib
```

Comment:
@seanpearsonuk, could you please have a look at this description and see if it seems fine?

cc. @mayankansys.